test/auro-accordion-group.test.js

```diff
@@ -0,0 +1,132 @@
+import { fixture, html, expect, elementUpdated } from '@open-wc/testing';
+import '../src/auro-accordion-group';
+
+describe('auro-accordion-group', () => {
```

Comment:
Need to add a test that only one accordion can be expanded in a group at a time. The icov report shows this code as not being tested

Reply:
Resolved